Documentation/ApiOverview/Events/Events/Backend/_ModifyRecordListTableActionsEvent/_MyEventListener.php

```diff
@@ -24,12 +24,9 @@
 )]
 final readonly class MyEventListener
 {
-    private LoggerInterface $logger;
-
-    public function __construct(LoggerInterface $logger)
-    {
-        $this->logger = $logger;
-    }
+    public function __construct(
+        private LoggerInterface $logger,
```

Comment:
Just search in core. They also use the "readonly" keyword

```suggestion
        private readonly LoggerInterface $logger,
```